src/libs/Network/SequentialQueue.js

```diff
@@ -114,9 +114,18 @@ function getCurrentRequest() {
     return currentRequest;
 }
 
+/**
+ * Returns a promise that resolves when the last request on the queue responds.
```

Comment:
Maybe this is a slightly more accurate description:
```suggestion
 * Returns a promise that resolves when the sequential queue is done processing all persisted write requests.
```

Because there is a case where this can be called, and there were no persisted write requests at all in the queue, and the promise is just immediately resolved. Right?

Reply:
I like it, I wasn't sure how to phrase it but I think you hit it on the nail